x/oracle/keeper/msg_server.go

```diff
@@ -108,8 +108,16 @@ func (ms msgServer) AggregateExchangeRateVote(
 		return nil, sdkerrors.Wrapf(types.ErrVerificationFailed, "must be given %s not %s", aggregatePrevote.Hash, hash)
 	}
 
+	// Filter out rates which aren't included in the AcceptList
+	filteredTuples := types.ExchangeRateTuples{}
```

Comment:
It was more straightforward to do it this way instead of removing elements from `exchangeRateTuples`, in case for some reason `exchangeRateTuples` is super long